# Russian message translation file for libpq
# Copyright (C) 2001-2016 PostgreSQL Global Development Group
# This file is distributed under the same license as the PostgreSQL package.
# Serguei A. Mokhov <mokhov@cs.concordia.ca>, 2001-2004.
# Oleg Bartunov <oleg@sai.msu.su>, 2005.
# Andrey Sudnik <sudnikand@yandex.ru>, 2010.
# Alexander Lakhin <exclusion@gmail.com>, 2012-2017, 2018, 2019, 2020, 2021.
# Maxim Yablokov <m.yablokov@postgrespro.ru>, 2021.
msgid ""
msgstr ""
"Project-Id-Version: libpq (PostgreSQL current)\n"
"Report-Msgid-Bugs-To: pgsql-bugs@lists.postgresql.org\n"
"POT-Creation-Date: 2022-02-07 11:21+0300\n"
"PO-Revision-Date: 2021-11-25 12:45+0300\n"
"Last-Translator: Alexander Lakhin <exclusion@gmail.com>\n"
"Language-Team: Russian <pgsql-ru-general@postgresql.org>\n"
"Language: ru\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n"
"%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"

#: fe-auth-scram.c:213
msgid "malformed SCRAM message (empty message)\n"
msgstr "неправильное сообщение SCRAM (пустое содержимое)\n"

#: fe-auth-scram.c:219
msgid "malformed SCRAM message (length mismatch)\n"
msgstr "неправильное сообщение SCRAM (некорректная длина)\n"

#: fe-auth-scram.c:263
msgid "could not verify server signature\n"
msgstr "не удалось проверить сигнатуру сервера\n"

#: fe-auth-scram.c:270
msgid "incorrect server signature\n"
msgstr "некорректная сигнатура сервера\n"

#: fe-auth-scram.c:279
msgid "invalid SCRAM exchange state\n"
msgstr "ошибочное состояние обмена SCRAM\n"

#: fe-auth-scram.c:306
#, c-format
msgid "malformed SCRAM message (attribute \"%c\" expected)\n"
msgstr "неправильное сообщение SCRAM (ожидался атрибут \"%c\")\n"

#: fe-auth-scram.c:315
#, c-format
msgid ""
"malformed SCRAM message (expected character \"=\" for attribute \"%c\")\n"
msgstr ""
"неправильное сообщение SCRAM (для атрибута \"%c\" ожидался символ \"=\")\n"

#: fe-auth-scram.c:356
msgid "could not generate nonce\n"
msgstr "не удалось сгенерировать разовый код\n"

#: fe-auth-scram.c:366 fe-auth-scram.c:441 fe-auth-scram.c:595
#: fe-auth-scram.c:616 fe-auth-scram.c:642 fe-auth-scram.c:657
#: fe-auth-scram.c:707 fe-auth-scram.c:746 fe-auth.c:290 fe-auth.c:362
#: fe-auth.c:398 fe-auth.c:615 fe-auth.c:774 fe-auth.c:1132 fe-auth.c:1282
#: fe-connect.c:911 fe-connect.c:1455 fe-connect.c:1624 fe-connect.c:2976
#: fe-connect.c:4706 fe-connect.c:4967 fe-connect.c:5086 fe-connect.c:5338
#: fe-connect.c:5419 fe-connect.c:5518 fe-connect.c:5774 fe-connect.c:5803
#: fe-connect.c:5875 fe-connect.c:5899 fe-connect.c:5917 fe-connect.c:6018
#: fe-connect.c:6027 fe-connect.c:6385 fe-connect.c:6535 fe-connect.c:6801
#: fe-exec.c:686 fe-exec.c:876 fe-exec.c:1223 fe-exec.c:3043 fe-exec.c:3226
#: fe-exec.c:3999 fe-exec.c:4164 fe-gssapi-common.c:111 fe-lobj.c:881
#: fe-protocol3.c:975 fe-protocol3.c:990 fe-protocol3.c:1023
#: fe-protocol3.c:1731 fe-secure-common.c:110 fe-secure-gssapi.c:504
#: fe-secure-openssl.c:440 fe-secure-openssl.c:1133
msgid "out of memory\n"
msgstr "нехватка памяти\n"

#: fe-auth-scram.c:374
msgid "could not encode nonce\n"
msgstr "не удалось оформить разовый код\n"

#: fe-auth-scram.c:563
msgid "could not calculate client proof\n"
msgstr "не удалось вычислить подтверждение клиента\n"

#: fe-auth-scram.c:579
msgid "could not encode client proof\n"
msgstr "не удалось закодировать подтверждение клиента\n"

#: fe-auth-scram.c:634
msgid "invalid SCRAM response (nonce mismatch)\n"
msgstr "неверный ответ SCRAM (несовпадение проверочного кода)\n"

#: fe-auth-scram.c:667
msgid "malformed SCRAM message (invalid salt)\n"
msgstr "неправильное сообщение SCRAM (некорректная соль)\n"

#: fe-auth-scram.c:681
msgid "malformed SCRAM message (invalid iteration count)\n"
msgstr "неправильное сообщение SCRAM (некорректное число итераций)\n"

#: fe-auth-scram.c:687
msgid "malformed SCRAM message (garbage at end of server-first-message)\n"
msgstr ""
"неправильное сообщение SCRAM (мусор в конце первого сообщения сервера)\n"

#: fe-auth-scram.c:723
#, c-format
msgid "error received from server in SCRAM exchange: %s\n"
msgstr "в ходе обмена SCRAM от сервера получена ошибка: %s\n"

#: fe-auth-scram.c:739
msgid "malformed SCRAM message (garbage at end of server-final-message)\n"
msgstr ""
"неправильное сообщение SCRAM (мусор в конце последнего сообщения сервера)\n"

#: fe-auth-scram.c:758
msgid "malformed SCRAM message (invalid server signature)\n"
msgstr "неправильное сообщение SCRAM (неверная сигнатура сервера)\n"

#: fe-auth.c:76
#, c-format
msgid "out of memory allocating GSSAPI buffer (%d)\n"
msgstr "недостаточно памяти для буфера GSSAPI (%d)\n"

#: fe-auth.c:131
msgid "GSSAPI continuation error"
msgstr "ошибка продолжения в GSSAPI"

#: fe-auth.c:158 fe-auth.c:391 fe-gssapi-common.c:98 fe-secure-common.c:98
msgid "host name must be specified\n"
msgstr "требуется указать имя сервера\n"

#: fe-auth.c:165
msgid "duplicate GSS authentication request\n"
msgstr "повторный запрос аутентификации GSS\n"

#: fe-auth.c:230
#, c-format
msgid "out of memory allocating SSPI buffer (%d)\n"
msgstr "недостаточно памяти для буфера SSPI (%d)\n"

#: fe-auth.c:278
msgid "SSPI continuation error"
msgstr "ошибка продолжения в SSPI"

#: fe-auth.c:351
msgid "duplicate SSPI authentication request\n"
msgstr "повторный запрос аутентификации SSPI\n"

#: fe-auth.c:377
msgid "could not acquire SSPI credentials"
msgstr "не удалось получить удостоверение SSPI"

#: fe-auth.c:433
msgid "channel binding required, but SSL not in use\n"
msgstr "требуется привязка каналов, но SSL не используется\n"

#: fe-auth.c:440
msgid "duplicate SASL authentication request\n"
msgstr "повторный запрос аутентификации SASL\n"

#: fe-auth.c:496
msgid "channel binding is required, but client does not support it\n"
msgstr "требуется привязка каналов, но клиент её не поддерживает\n"

#: fe-auth.c:513
msgid ""
"server offered SCRAM-SHA-256-PLUS authentication over a non-SSL connection\n"
msgstr ""
"сервер предложил аутентификацию SCRAM-SHA-256-PLUS для соединения, не "
"защищённого SSL\n"

#: fe-auth.c:525
msgid "none of the server's SASL authentication mechanisms are supported\n"
msgstr ""
"ни один из серверных механизмов аутентификации SASL не поддерживается\n"

#: fe-auth.c:533
msgid ""
"channel binding is required, but server did not offer an authentication "
"method that supports channel binding\n"
msgstr ""
"требуется привязка каналов, но сервер не предложил поддерживающий её метод "
"аутентификации\n"

#: fe-auth.c:639
#, c-format
msgid "out of memory allocating SASL buffer (%d)\n"
msgstr "недостаточно памяти для буфера SASL (%d)\n"

#: fe-auth.c:664
msgid ""
"AuthenticationSASLFinal received from server, but SASL authentication was "
"not completed\n"
msgstr ""
"c сервера получено сообщение AuthenticationSASLFinal, но аутентификация SASL "
"ещё не завершена\n"

#: fe-auth.c:741
msgid "SCM_CRED authentication method not supported\n"
msgstr "аутентификация SCM_CRED не поддерживается\n"

#: fe-auth.c:836
msgid ""
"channel binding required, but server authenticated client without channel "
"binding\n"
msgstr ""
"требуется привязка каналов, но сервер аутентифицировал клиента без привязки\n"

#: fe-auth.c:842
msgid ""
"channel binding required but not supported by server's authentication "
"request\n"
msgstr ""
"требуется привязка каналов, но она не поддерживается при том запросе "
"аутентификации, который передал сервер\n"

#: fe-auth.c:877
msgid "Kerberos 4 authentication not supported\n"
msgstr "аутентификация Kerberos 4 не поддерживается\n"

#: fe-auth.c:882
msgid "Kerberos 5 authentication not supported\n"
msgstr "аутентификация Kerberos 5 не поддерживается\n"

#: fe-auth.c:953
msgid "GSSAPI authentication not supported\n"
msgstr "аутентификация через GSSAPI не поддерживается\n"

#: fe-auth.c:985
msgid "SSPI authentication not supported\n"
msgstr "аутентификация через SSPI не поддерживается\n"

#: fe-auth.c:993
msgid "Crypt authentication not supported\n"
msgstr "аутентификация Crypt не поддерживается\n"

#: fe-auth.c:1060
#, c-format
msgid "authentication method %u not supported\n"
msgstr "метод аутентификации %u не поддерживается\n"

#: fe-auth.c:1107
#, c-format
msgid "user name lookup failure: error code %lu\n"
msgstr "распознать имя пользователя не удалось (код ошибки: %lu)\n"

#: fe-auth.c:1117 fe-connect.c:2851
#, c-format
msgid "could not look up local user ID %d: %s\n"
msgstr "найти локального пользователя по идентификатору (%d) не удалось: %s\n"

#: fe-auth.c:1122 fe-connect.c:2856
#, c-format
msgid "local user with ID %d does not exist\n"
msgstr "локальный пользователь с ID %d не существует\n"

#: fe-auth.c:1226
msgid "unexpected shape of result set returned for SHOW\n"
msgstr "неожиданная форма набора результатов, возвращённого для SHOW\n"

#: fe-auth.c:1235
msgid "password_encryption value too long\n"
msgstr "слишком длинное значение password_encryption\n"

#: fe-auth.c:1275
#, c-format
msgid "unrecognized password encryption algorithm \"%s\"\n"
msgstr "нераспознанный алгоритм шифрования пароля \"%s\"\n"

#: fe-connect.c:1094
#, c-format
msgid "could not match %d host names to %d hostaddr values\n"
msgstr "не удалось сопоставить имена узлов (%d) со значениями hostaddr (%d)\n"

#: fe-connect.c:1175
#, c-format
msgid "could not match %d port numbers to %d hosts\n"
msgstr "не удалось сопоставить номера портов (%d) с узлами (%d)\n"

#: fe-connect.c:1268 fe-connect.c:1294 fe-connect.c:1336 fe-connect.c:1345
#: fe-connect.c:1378 fe-connect.c:1422
#, c-format
msgid "invalid %s value: \"%s\"\n"
msgstr "неверное значение %s: \"%s\"\n"

#: fe-connect.c:1315
#, c-format
msgid "sslmode value \"%s\" invalid when SSL support is not compiled in\n"
msgstr "значение sslmode \"%s\" недопустимо для сборки без поддержки SSL\n"

#: fe-connect.c:1363
msgid "invalid SSL protocol version range\n"
msgstr "неверный диапазон версий протокола SSL\n"

#: fe-connect.c:1388
#, c-format
msgid ""
"gssencmode value \"%s\" invalid when GSSAPI support is not compiled in\n"
msgstr ""
"значение gssencmode \"%s\" недопустимо для сборки без поддержки GSSAPI\n"

#: fe-connect.c:1648
#, c-format
msgid "could not set socket to TCP no delay mode: %s\n"
msgstr "не удалось перевести сокет в режим TCP-передачи без задержки: %s\n"

#: fe-connect.c:1710
#, c-format
msgid "connection to server on socket \"%s\" failed: "
msgstr "подключиться к серверу через сокет \"%s\" не удалось: "

#: fe-connect.c:1737
#, c-format
msgid "connection to server at \"%s\" (%s), port %s failed: "
msgstr "подключиться к серверу \"%s\" (%s), порту %s не удалось: "

#: fe-connect.c:1742
#, c-format
msgid "connection to server at \"%s\", port %s failed: "
msgstr "подключиться к серверу \"%s\", порту %s не удалось: "

#: fe-connect.c:1767
msgid ""
"\tIs the server running locally and accepting connections on that socket?\n"
msgstr ""
"\tСервер действительно работает локально и принимает подключения через этот "
"сокет?\n"

#: fe-connect.c:1771
msgid ""
"\tIs the server running on that host and accepting TCP/IP connections?\n"
msgstr ""
"\tСервер действительно работает по данному адресу и принимает TCP-"
"соединения?\n"

#: fe-connect.c:1835
#, c-format
msgid "invalid integer value \"%s\" for connection option \"%s\"\n"
msgstr ""
"неверное целочисленное значение \"%s\" для параметра соединения \"%s\"\n"

#: fe-connect.c:1865 fe-connect.c:1900 fe-connect.c:1936 fe-connect.c:2025
#: fe-connect.c:2639
#, c-format
msgid "%s(%s) failed: %s\n"
msgstr "ошибка в %s(%s): %s\n"

#: fe-connect.c:1990
#, c-format
msgid "%s(%s) failed: error code %d\n"
msgstr "ошибка в %s(%s): код ошибки %d\n"

#: fe-connect.c:2305
msgid "invalid connection state, probably indicative of memory corruption\n"
msgstr "неверное состояние соединения - возможно разрушение памяти\n"

#: fe-connect.c:2384
#, c-format
msgid "invalid port number: \"%s\"\n"
msgstr "неверный номер порта: \"%s\"\n"

#: fe-connect.c:2400
#, c-format
msgid "could not translate host name \"%s\" to address: %s\n"
msgstr "преобразовать имя \"%s\" в адрес не удалось: %s\n"

#: fe-connect.c:2413
#, c-format
msgid "could not parse network address \"%s\": %s\n"
msgstr "не удалось разобрать сетевой адрес \"%s\": %s\n"

#: fe-connect.c:2426
#, c-format
msgid "Unix-domain socket path \"%s\" is too long (maximum %d bytes)\n"
msgstr "длина пути Unix-сокета \"%s\" превышает предел (%d байт)\n"

#: fe-connect.c:2441
#, c-format
msgid "could not translate Unix-domain socket path \"%s\" to address: %s\n"
msgstr "преобразовать путь Unix-сокета \"%s\" в адрес не удалось: %s\n"

#: fe-connect.c:2567
#, c-format
msgid "could not create socket: %s\n"
msgstr "не удалось создать сокет: %s\n"

#: fe-connect.c:2598
#, c-format
msgid "could not set socket to nonblocking mode: %s\n"
msgstr "не удалось перевести сокет в неблокирующий режим: %s\n"

#: fe-connect.c:2608
#, c-format
msgid "could not set socket to close-on-exec mode: %s\n"
msgstr ""
"не удалось перевести сокет в режим закрытия при выполнении (close-on-exec): "
"%s\n"

#: fe-connect.c:2626
msgid "keepalives parameter must be an integer\n"
msgstr "параметр keepalives должен быть целым числом\n"

#: fe-connect.c:2767
#, c-format
msgid "could not get socket error status: %s\n"
msgstr "не удалось получить статус ошибки сокета: %s\n"

#: fe-connect.c:2795
#, c-format
msgid "could not get client address from socket: %s\n"
msgstr "не удалось получить адрес клиента из сокета: %s\n"

#: fe-connect.c:2837
msgid "requirepeer parameter is not supported on this platform\n"
msgstr "параметр requirepeer не поддерживается в этой ОС\n"

#: fe-connect.c:2840
#, c-format
msgid "could not get peer credentials: %s\n"
msgstr "не удалось получить учётные данные сервера: %s\n"

#: fe-connect.c:2864
#, c-format
msgid "requirepeer specifies \"%s\", but actual peer user name is \"%s\"\n"
msgstr ""
"requirepeer допускает подключение только к \"%s\", но сервер работает под "
"именем \"%s\"\n"

#: fe-connect.c:2904
#, c-format
msgid "could not send GSSAPI negotiation packet: %s\n"
msgstr "не удалось отправить пакет согласования GSSAPI: %s\n"

#: fe-connect.c:2916
msgid ""
"GSSAPI encryption required but was impossible (possibly no credential cache, "
"no server support, or using a local socket)\n"
msgstr ""
"затребовано шифрование GSSAPI, но это требование невыполнимо (возможно, "
"отсутствует кеш учётных данных, нет поддержки на сервере или используется "
"локальный сокет)\n"

#: fe-connect.c:2958
#, c-format
msgid "could not send SSL negotiation packet: %s\n"
msgstr "не удалось отправить пакет согласования SSL: %s\n"

#: fe-connect.c:2989
#, c-format
msgid "could not send startup packet: %s\n"
msgstr "не удалось отправить стартовый пакет: %s\n"

#: fe-connect.c:3065
msgid "server does not support SSL, but SSL was required\n"
msgstr "затребовано подключение через SSL, но сервер не поддерживает SSL\n"

#: fe-connect.c:3092
#, c-format
msgid "received invalid response to SSL negotiation: %c\n"
msgstr "получен неверный ответ при согласовании SSL: %c\n"

#: fe-connect.c:3113
msgid "received unencrypted data after SSL response\n"
msgstr "после ответа SSL получены незашифрованные данные\n"

#: fe-connect.c:3194
msgid "server doesn't support GSSAPI encryption, but it was required\n"
msgstr "затребовано шифрование GSSAPI, но сервер его не поддерживает\n"

#: fe-connect.c:3206
#, c-format
msgid "received invalid response to GSSAPI negotiation: %c\n"
msgstr "получен неверный ответ при согласовании GSSAPI: %c\n"

#: fe-connect.c:3225
msgid "received unencrypted data after GSSAPI encryption response\n"
msgstr ""
"после ответа на запрос шифрования GSSAPI получены незашифрованные данные\n"

#: fe-connect.c:3285 fe-connect.c:3310
#, c-format
msgid "expected authentication request from server, but received %c\n"
msgstr "ожидался запрос аутентификации от сервера, но получено: %c\n"

#: fe-connect.c:3517
msgid "unexpected message from server during startup\n"
msgstr "неожиданное сообщение от сервера в начале работы\n"

#: fe-connect.c:3609
msgid "session is read-only\n"
msgstr "сеанс не допускает запись\n"

#: fe-connect.c:3612
msgid "session is not read-only\n"
msgstr "сеанс допускает запись\n"

#: fe-connect.c:3666
msgid "server is in hot standby mode\n"
msgstr "сервер работает в режиме горячего резерва\n"

#: fe-connect.c:3669
msgid "server is not in hot standby mode\n"
msgstr "сервер работает не в режиме горячего резерва\n"

#: fe-connect.c:3787 fe-connect.c:3839
#, c-format
msgid "\"%s\" failed\n"
msgstr "выполнить \"%s\" не удалось\n"

#: fe-connect.c:3853
#, c-format
msgid "invalid connection state %d, probably indicative of memory corruption\n"
msgstr "неверное состояние соединения %d - возможно разрушение памяти\n"

#: fe-connect.c:4299 fe-connect.c:4359
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_CONNRESET event\n"
msgstr "ошибка в PGEventProc \"%s\" при обработке события PGEVT_CONNRESET\n"

#: fe-connect.c:4719
#, c-format
msgid "invalid LDAP URL \"%s\": scheme must be ldap://\n"
msgstr "некорректный адрес LDAP \"%s\": схема должна быть ldap://\n"

#: fe-connect.c:4734
#, c-format
msgid "invalid LDAP URL \"%s\": missing distinguished name\n"
msgstr "некорректный адрес LDAP \"%s\": отсутствует уникальное имя\n"

#: fe-connect.c:4746 fe-connect.c:4804
#, c-format
msgid "invalid LDAP URL \"%s\": must have exactly one attribute\n"
msgstr "некорректный адрес LDAP \"%s\": должен быть только один атрибут\n"

#: fe-connect.c:4758 fe-connect.c:4820
#, c-format
msgid "invalid LDAP URL \"%s\": must have search scope (base/one/sub)\n"
msgstr ""
"некорректный адрес LDAP \"%s\": не указана область поиска (base/one/sub)\n"

#: fe-connect.c:4770
#, c-format
msgid "invalid LDAP URL \"%s\": no filter\n"
msgstr "некорректный адрес LDAP \"%s\": нет фильтра\n"

#: fe-connect.c:4792
#, c-format
msgid "invalid LDAP URL \"%s\": invalid port number\n"
msgstr "некорректный адрес LDAP \"%s\": неверный номер порта\n"

#: fe-connect.c:4830
msgid "could not create LDAP structure\n"
msgstr "не удалось создать структуру LDAP\n"

#: fe-connect.c:4906
#, c-format
msgid "lookup on LDAP server failed: %s\n"
msgstr "ошибка поиска на сервере LDAP: %s\n"

#: fe-connect.c:4917
msgid "more than one entry found on LDAP lookup\n"
msgstr "при поиске LDAP найдено более одного вхождения\n"

#: fe-connect.c:4918 fe-connect.c:4930
msgid "no entry found on LDAP lookup\n"
msgstr "при поиске LDAP ничего не найдено\n"

#: fe-connect.c:4941 fe-connect.c:4954
msgid "attribute has no values on LDAP lookup\n"
msgstr "атрибут не содержит значений при поиске LDAP\n"

#: fe-connect.c:5006 fe-connect.c:5025 fe-connect.c:5557
#, c-format
msgid "missing \"=\" after \"%s\" in connection info string\n"
msgstr "в строке соединения нет \"=\" после \"%s\"\n"

#: fe-connect.c:5098 fe-connect.c:5742 fe-connect.c:6518
#, c-format
msgid "invalid connection option \"%s\"\n"
msgstr "неверный параметр соединения \"%s\"\n"

#: fe-connect.c:5114 fe-connect.c:5606
msgid "unterminated quoted string in connection info string\n"
msgstr "в строке соединения не хватает закрывающей кавычки\n"

#: fe-connect.c:5195
#, c-format
msgid "definition of service \"%s\" not found\n"
msgstr "определение службы \"%s\" не найдено\n"

#: fe-connect.c:5221
#, c-format
msgid "service file \"%s\" not found\n"
msgstr "файл определений служб \"%s\" не найден\n"

#: fe-connect.c:5235
#, c-format
msgid "line %d too long in service file \"%s\"\n"
msgstr "слишком длинная строка (%d) в файле определений служб \"%s\"\n"

#: fe-connect.c:5306 fe-connect.c:5350
#, c-format
msgid "syntax error in service file \"%s\", line %d\n"
msgstr "синтаксическая ошибка в файле определения служб \"%s\" (строка %d)\n"

#: fe-connect.c:5317
#, c-format
msgid ""
"nested service specifications not supported in service file \"%s\", line %d\n"
msgstr ""
"рекурсивные определения служб не поддерживаются (файл определения служб \"%s"
"\", строка %d)\n"

#: fe-connect.c:6038
#, c-format
msgid "invalid URI propagated to internal parser routine: \"%s\"\n"
msgstr "во внутреннюю процедуру разбора строки передан ошибочный URI: \"%s\"\n"

#: fe-connect.c:6115
#, c-format
msgid ""
"end of string reached when looking for matching \"]\" in IPv6 host address "
"in URI: \"%s\"\n"
msgstr "URI не содержит символ \"]\" после адреса IPv6: \"%s\"\n"

#: fe-connect.c:6122
#, c-format
msgid "IPv6 host address may not be empty in URI: \"%s\"\n"
msgstr "IPv6, содержащийся в URI, не может быть пустым: \"%s\"\n"

#: fe-connect.c:6137
#, c-format
msgid ""
"unexpected character \"%c\" at position %d in URI (expected \":\" or \"/\"): "
"\"%s\"\n"
msgstr ""
"неожиданный символ \"%c\" в позиции %d в URI (ожидалось \":\" или \"/\"): "
"\"%s\"\n"

#: fe-connect.c:6267
#, c-format
msgid "extra key/value separator \"=\" in URI query parameter: \"%s\"\n"
msgstr "лишний разделитель ключа/значения \"=\" в параметрах URI: \"%s\"\n"

#: fe-connect.c:6287
#, c-format
msgid "missing key/value separator \"=\" in URI query parameter: \"%s\"\n"
msgstr "в параметрах URI не хватает разделителя ключа/значения \"=\": \"%s\"\n"

#: fe-connect.c:6339
#, c-format
msgid "invalid URI query parameter: \"%s\"\n"
msgstr "неверный параметр в URI: \"%s\"\n"

#: fe-connect.c:6413
#, c-format
msgid "invalid percent-encoded token: \"%s\"\n"
msgstr "неверный символ, закодированный с %%: \"%s\"\n"

#: fe-connect.c:6423
#, c-format
msgid "forbidden value %%00 in percent-encoded value: \"%s\"\n"
msgstr "недопустимое значение %%00 для символа, закодированного с %%: \"%s\"\n"

#: fe-connect.c:6793
msgid "connection pointer is NULL\n"
msgstr "нулевой указатель соединения\n"

#: fe-connect.c:7081
#, c-format
msgid "WARNING: password file \"%s\" is not a plain file\n"
msgstr "ПРЕДУПРЕЖДЕНИЕ: файл паролей \"%s\" - не обычный файл\n"

#: fe-connect.c:7090
#, c-format
msgid ""
"WARNING: password file \"%s\" has group or world access; permissions should "
"be u=rw (0600) or less\n"
msgstr ""
"ПРЕДУПРЕЖДЕНИЕ: к файлу паролей \"%s\" имеют доступ все или группа; права "
"должны быть u=rw (0600) или более ограниченные\n"

#: fe-connect.c:7198
#, c-format
msgid "password retrieved from file \"%s\"\n"
msgstr "пароль получен из файла \"%s\"\n"

#: fe-exec.c:449 fe-exec.c:3300
#, c-format
msgid "row number %d is out of range 0..%d"
msgstr "номер записи %d вне диапазона 0..%d"

#: fe-exec.c:510 fe-protocol3.c:219 fe-protocol3.c:244 fe-protocol3.c:273
#: fe-protocol3.c:291 fe-protocol3.c:371 fe-protocol3.c:743
msgid "out of memory"
msgstr "нехватка памяти"

#: fe-exec.c:511 fe-protocol3.c:1939
#, c-format
msgid "%s"
msgstr "%s"

#: fe-exec.c:792
msgid "write to server failed\n"
msgstr "ошибка при передаче данных серверу\n"

#: fe-exec.c:864
msgid "NOTICE"
msgstr "ЗАМЕЧАНИЕ"

#: fe-exec.c:922
msgid "PGresult cannot support more than INT_MAX tuples"
msgstr "PGresult не может вместить больше чем INT_MAX кортежей"

#: fe-exec.c:934
msgid "size_t overflow"
msgstr "переполнение size_t"

#: fe-exec.c:1349 fe-exec.c:1454 fe-exec.c:1503
msgid "command string is a null pointer\n"
msgstr "указатель на командную строку нулевой\n"

#: fe-exec.c:1460 fe-exec.c:1509 fe-exec.c:1605
#, c-format
msgid "number of parameters must be between 0 and %d\n"
msgstr "число параметров должно быть от 0 до %d\n"

#: fe-exec.c:1497 fe-exec.c:1599
msgid "statement name is a null pointer\n"
msgstr "указатель на имя оператора нулевой\n"

#: fe-exec.c:1641 fe-exec.c:3153
msgid "no connection to the server\n"
msgstr "нет соединения с сервером\n"

#: fe-exec.c:1650 fe-exec.c:3162
msgid "another command is already in progress\n"
msgstr "уже выполняется другая команда\n"

#: fe-exec.c:1679
msgid "cannot queue commands during COPY\n"
msgstr "во время COPY нельзя добавлять команды в очередь\n"

#: fe-exec.c:1797
msgid "length must be given for binary parameter\n"
msgstr "для двоичного параметра должна быть указана длина\n"

#: fe-exec.c:2117
#, c-format
msgid "unexpected asyncStatus: %d\n"
msgstr "неожиданный asyncStatus: %d\n"

#: fe-exec.c:2137
#, c-format
msgid "PGEventProc \"%s\" failed during PGEVT_RESULTCREATE event\n"
msgstr "ошибка в PGEventProc \"%s\" при обработке события PGEVT_RESULTCREATE\n"

#: fe-exec.c:2285
msgid ""
"synchronous command execution functions are not allowed in pipeline mode\n"
msgstr ""
"функции синхронного выполнения команд не допускаются в конвейерном режиме\n"

#: fe-exec.c:2307
msgid "COPY terminated by new PQexec"
msgstr "операция COPY прервана вызовом PQexec"

#: fe-exec.c:2324
msgid "PQexec not allowed during COPY BOTH\n"
msgstr "вызов PQexec не допускается в процессе COPY BOTH\n"

#: fe-exec.c:2552 fe-exec.c:2608 fe-exec.c:2677 fe-protocol3.c:1870
msgid "no COPY in progress\n"
msgstr "операция COPY не выполняется\n"

#: fe-exec.c:2854
msgid "PQfn not allowed in pipeline mode\n"
msgstr "PQfn не допускается в конвейерном режиме\n"

#: fe-exec.c:2862
msgid "connection in wrong state\n"
msgstr "соединение в неправильном состоянии\n"

#: fe-exec.c:2906
msgid "cannot enter pipeline mode, connection not idle\n"
msgstr "перейти в конвейерный режиме нельзя, соединение не простаивает\n"

#: fe-exec.c:2940 fe-exec.c:2957
msgid "cannot exit pipeline mode with uncollected results\n"
msgstr "выйти из конвейерного режима нельзя, не собрав все результаты\n"

#: fe-exec.c:2945
msgid "cannot exit pipeline mode while busy\n"
msgstr "выйти из конвейерного режима в занятом состоянии нельзя\n"

#: fe-exec.c:3087
msgid "cannot send pipeline when not in pipeline mode\n"
msgstr "отправить конвейер, не перейдя в конвейерный режим, нельзя\n"

#: fe-exec.c:3189
msgid "invalid ExecStatusType code"
msgstr "неверный код ExecStatusType"

#: fe-exec.c:3216
msgid "PGresult is not an error result\n"
msgstr "В PGresult не передан результат ошибки\n"

#: fe-exec.c:3284 fe-exec.c:3307
#, c-format
msgid "column number %d is out of range 0..%d"
msgstr "номер столбца %d вне диапазона 0..%d"

#: fe-exec.c:3322
#, c-format
msgid "parameter number %d is out of range 0..%d"
msgstr "номер параметра %d вне диапазона 0..%d"

#: fe-exec.c:3632
#, c-format
msgid "could not interpret result from server: %s"
msgstr "не удалось интерпретировать ответ сервера: %s"

#: fe-exec.c:3892 fe-exec.c:3981
msgid "incomplete multibyte character\n"
msgstr "неполный многобайтный символ\n"

#: fe-gssapi-common.c:124
msgid "GSSAPI name import error"
msgstr "ошибка импорта имени в GSSAPI"

#: fe-lobj.c:145 fe-lobj.c:210 fe-lobj.c:403 fe-lobj.c:494 fe-lobj.c:568
#: fe-lobj.c:969 fe-lobj.c:977 fe-lobj.c:985 fe-lobj.c:993 fe-lobj.c:1001
#: fe-lobj.c:1009 fe-lobj.c:1017 fe-lobj.c:1025
#, c-format
msgid "cannot determine OID of function %s\n"
msgstr "определить OID функции %s нельзя\n"

#: fe-lobj.c:162
msgid "argument of lo_truncate exceeds integer range\n"
msgstr "аргумент lo_truncate не умещается в обычном целом\n"

#: fe-lobj.c:266
msgid "argument of lo_read exceeds integer range\n"
msgstr "аргумент lo_read не умещается в обычном целом\n"

#: fe-lobj.c:318
msgid "argument of lo_write exceeds integer range\n"
msgstr "аргумент lo_write не умещается в обычном целом\n"

#: fe-lobj.c:678 fe-lobj.c:789
#, c-format
msgid "could not open file \"%s\": %s\n"
msgstr "не удалось открыть файл \"%s\": %s\n"

#: fe-lobj.c:734
#, c-format
msgid "could not read from file \"%s\": %s\n"
msgstr "не удалось прочитать файл \"%s\": %s\n"

#: fe-lobj.c:810 fe-lobj.c:834
#, c-format
msgid "could not write to file \"%s\": %s\n"
msgstr "не удалось записать файл \"%s\": %s\n"

#: fe-lobj.c:920
msgid "query to initialize large object functions did not return data\n"
msgstr "запрос инициализации функций для больших объектов не вернул данные\n"

#: fe-misc.c:242
#, c-format
msgid "integer of size %lu not supported by pqGetInt"
msgstr "функция pqGetInt не поддерживает integer размером %lu байт"

#: fe-misc.c:275
#, c-format
msgid "integer of size %lu not supported by pqPutInt"
msgstr "функция pqPutInt не поддерживает integer размером %lu байт"

#: fe-misc.c:576 fe-misc.c:822
msgid "connection not open\n"
msgstr "соединение не открыто\n"

#: fe-misc.c:755 fe-secure-openssl.c:209 fe-secure-openssl.c:316
#: fe-secure.c:260 fe-secure.c:373
msgid ""
"server closed the connection unexpectedly\n"
"\tThis probably means the server terminated abnormally\n"
"\tbefore or while processing the request.\n"
msgstr ""
"сервер неожиданно закрыл соединение\n"
"\tСкорее всего сервер прекратил работу из-за сбоя\n"
"\tдо или в процессе выполнения запроса.\n"

#: fe-misc.c:1015
msgid "timeout expired\n"
msgstr "тайм-аут\n"

#: fe-misc.c:1060
msgid "invalid socket\n"
msgstr "неверный сокет\n"

#: fe-misc.c:1083
#, c-format
msgid "%s() failed: %s\n"
msgstr "ошибка в %s(): %s\n"

#: fe-protocol3.c:196
#, c-format
msgid "message type 0x%02x arrived from server while idle"
msgstr "от сервера во время простоя получено сообщение типа 0x%02x"

#: fe-protocol3.c:403
msgid ""
"server sent data (\"D\" message) without prior row description (\"T\" "
"message)\n"
msgstr ""
"сервер отправил данные (сообщение \"D\") без предварительного описания "
"строки (сообщение \"T\")\n"

#: fe-protocol3.c:446
#, c-format
msgid "unexpected response from server; first received character was \"%c\"\n"
msgstr "неожиданный ответ сервера; первый полученный символ: \"%c\"\n"

#: fe-protocol3.c:471
#, c-format
msgid "message contents do not agree with length in message type \"%c\"\n"
msgstr "содержимое не соответствует длине в сообщении типа \"%c\"\n"

#: fe-protocol3.c:491
#, c-format
msgid "lost synchronization with server: got message type \"%c\", length %d\n"
msgstr ""
"потеряна синхронизация с сервером: получено сообщение типа \"%c\", длина %d\n"

#: fe-protocol3.c:543 fe-protocol3.c:583
msgid "insufficient data in \"T\" message"
msgstr "недостаточно данных в сообщении \"T\""

#: fe-protocol3.c:654 fe-protocol3.c:860
msgid "out of memory for query result"
msgstr "недостаточно памяти для результата запроса"

#: fe-protocol3.c:723
msgid "insufficient data in \"t\" message"
msgstr "недостаточно данных в сообщении \"t\""

#: fe-protocol3.c:782 fe-protocol3.c:814 fe-protocol3.c:832
msgid "insufficient data in \"D\" message"
msgstr "недостаточно данных в сообщении \"D\""

#: fe-protocol3.c:788
msgid "unexpected field count in \"D\" message"
msgstr "неверное число полей в сообщении \"D\""

#: fe-protocol3.c:1036
msgid "no error message available\n"
msgstr "нет сообщения об ошибке\n"

#. translator: %s represents a digit string
#: fe-protocol3.c:1084 fe-protocol3.c:1103
#, c-format
msgid " at character %s"
msgstr " символ %s"

#: fe-protocol3.c:1116
#, c-format
msgid "DETAIL:  %s\n"
msgstr "ПОДРОБНОСТИ:  %s\n"

#: fe-protocol3.c:1119
#, c-format
msgid "HINT:  %s\n"
msgstr "ПОДСКАЗКА:  %s\n"

#: fe-protocol3.c:1122
#, c-format
msgid "QUERY:  %s\n"
msgstr "ЗАПРОС:  %s\n"

#: fe-protocol3.c:1129
#, c-format
msgid "CONTEXT:  %s\n"
msgstr "КОНТЕКСТ:  %s\n"

#: fe-protocol3.c:1138
#, c-format
msgid "SCHEMA NAME:  %s\n"
msgstr "СХЕМА:  %s\n"

#: fe-protocol3.c:1142
#, c-format
msgid "TABLE NAME:  %s\n"
msgstr "ТАБЛИЦА:  %s\n"

#: fe-protocol3.c:1146
#, c-format
msgid "COLUMN NAME:  %s\n"
msgstr "СТОЛБЕЦ:  %s\n"

#: fe-protocol3.c:1150
#, c-format
msgid "DATATYPE NAME:  %s\n"
msgstr "ТИП ДАННЫХ:  %s\n"

#: fe-protocol3.c:1154
#, c-format
msgid "CONSTRAINT NAME:  %s\n"
msgstr "ОГРАНИЧЕНИЕ:  %s\n"

#: fe-protocol3.c:1166
msgid "LOCATION:  "
msgstr "ПОЛОЖЕНИЕ:  "

#: fe-protocol3.c:1168
#, c-format
msgid "%s, "
msgstr "%s, "

#: fe-protocol3.c:1170
#, c-format
msgid "%s:%s"
msgstr "%s:%s"

#: fe-protocol3.c:1365
#, c-format
msgid "LINE %d: "
msgstr "СТРОКА %d: "

#: fe-protocol3.c:1764
msgid "PQgetline: not doing text COPY OUT\n"
msgstr "PQgetline можно вызывать только во время COPY OUT с текстом\n"

#: fe-protocol3.c:2130
#, c-format
msgid "protocol error: id=0x%x\n"
msgstr "ошибка протокола: id=0x%x\n"

#: fe-secure-common.c:124
msgid "SSL certificate's name contains embedded null\n"
msgstr "имя в SSL-сертификате включает нулевой байт\n"

#: fe-secure-common.c:171
msgid "host name must be specified for a verified SSL connection\n"
msgstr "для проверенного SSL-соединения требуется указать имя узла\n"

#: fe-secure-common.c:196
#, c-format
msgid "server certificate for \"%s\" does not match host name \"%s\"\n"
msgstr ""
"серверный сертификат для \"%s\" не соответствует имени сервера \"%s\"\n"

#: fe-secure-common.c:202
msgid "could not get server's host name from server certificate\n"
msgstr "не удалось получить имя сервера из сертификата\n"

#: fe-secure-gssapi.c:201
msgid "GSSAPI wrap error"
msgstr "ошибка обёртывания сообщения в GSSAPI"

#: fe-secure-gssapi.c:209
msgid "outgoing GSSAPI message would not use confidentiality\n"
msgstr "исходящее сообщение GSSAPI не будет защищено\n"

#: fe-secure-gssapi.c:217
#, c-format
msgid "client tried to send oversize GSSAPI packet (%zu > %zu)\n"
msgstr "клиент попытался передать чрезмерно большой пакет GSSAPI (%zu > %zu)\n"

#: fe-secure-gssapi.c:354 fe-secure-gssapi.c:596
#, c-format
msgid "oversize GSSAPI packet sent by the server (%zu > %zu)\n"
msgstr "сервер передал чрезмерно большой пакет GSSAPI (%zu > %zu)\n"

#: fe-secure-gssapi.c:393
msgid "GSSAPI unwrap error"
msgstr "ошибка развёртывания сообщения в GSSAPI"

#: fe-secure-gssapi.c:403
msgid "incoming GSSAPI message did not use confidentiality\n"
msgstr "входящее сообщение GSSAPI не защищено\n"

#: fe-secure-gssapi.c:642
msgid "could not initiate GSSAPI security context"
msgstr "не удалось инициализировать контекст безопасности GSSAPI"

#: fe-secure-gssapi.c:670
msgid "GSSAPI size check error"
msgstr "ошибка проверки размера в GSSAPI"

#: fe-secure-gssapi.c:681
msgid "GSSAPI context establishment error"
msgstr "ошибка установления контекста в GSSAPI"

#: fe-secure-openssl.c:214 fe-secure-openssl.c:321 fe-secure-openssl.c:1333
#, c-format
msgid "SSL SYSCALL error: %s\n"
msgstr "ошибка SSL SYSCALL: %s\n"

#: fe-secure-openssl.c:221 fe-secure-openssl.c:328 fe-secure-openssl.c:1337
msgid "SSL SYSCALL error: EOF detected\n"
msgstr "ошибка SSL SYSCALL: конец файла (EOF)\n"

#: fe-secure-openssl.c:232 fe-secure-openssl.c:339 fe-secure-openssl.c:1346
#, c-format
msgid "SSL error: %s\n"
msgstr "ошибка SSL: %s\n"

#: fe-secure-openssl.c:247 fe-secure-openssl.c:354
msgid "SSL connection has been closed unexpectedly\n"
msgstr "SSL-соединение было неожиданно закрыто\n"

#: fe-secure-openssl.c:253 fe-secure-openssl.c:360 fe-secure-openssl.c:1396
#, c-format
msgid "unrecognized SSL error code: %d\n"
msgstr "нераспознанный код ошибки SSL: %d\n"

#: fe-secure-openssl.c:400
msgid "could not determine server certificate signature algorithm\n"
msgstr "не удалось определить алгоритм подписи сертификата сервера\n"

#: fe-secure-openssl.c:421
#, c-format
msgid "could not find digest for NID %s\n"
msgstr "не удалось найти алгоритм хеширования по NID %s\n"

#: fe-secure-openssl.c:431
msgid "could not generate peer certificate hash\n"
msgstr "не удалось сгенерировать хеш сертификата сервера\n"

#: fe-secure-openssl.c:488
msgid "SSL certificate's name entry is missing\n"
msgstr "запись имени в SSL-сертификате отсутствует\n"

#: fe-secure-openssl.c:822
#, c-format
msgid "could not create SSL context: %s\n"
msgstr "не удалось создать контекст SSL: %s\n"

#: fe-secure-openssl.c:861
#, c-format
msgid "invalid value \"%s\" for minimum SSL protocol version\n"
msgstr "неверное значение \"%s\" для минимальной версии протокола SSL\n"

#: fe-secure-openssl.c:872
#, c-format
msgid "could not set minimum SSL protocol version: %s\n"
msgstr "не удалось задать минимальную версию протокола SSL: %s\n"

#: fe-secure-openssl.c:890
#, c-format
msgid "invalid value \"%s\" for maximum SSL protocol version\n"
msgstr "неверное значение \"%s\" для максимальной версии протокола SSL\n"

#: fe-secure-openssl.c:901
#, c-format
msgid "could not set maximum SSL protocol version: %s\n"
msgstr "не удалось задать максимальную версию протокола SSL: %s\n"

#: fe-secure-openssl.c:937
#, c-format
msgid "could not read root certificate file \"%s\": %s\n"
msgstr "не удалось прочитать файл корневых сертификатов \"%s\": %s\n"

#: fe-secure-openssl.c:990
msgid ""
"could not get home directory to locate root certificate file\n"
"Either provide the file or change sslmode to disable server certificate "
"verification.\n"
msgstr ""
"не удалось получить домашний каталог для поиска файла корневых сертификатов\n"
"Укажите полный путь к файлу или отключите проверку сертификата сервера, "
"изменив sslmode.\n"

#: fe-secure-openssl.c:994
#, c-format
msgid ""
"root certificate file \"%s\" does not exist\n"
"Either provide the file or change sslmode to disable server certificate "
"verification.\n"
msgstr ""
"файл корневых сертификатов \"%s\" не существует\n"
"Укажите полный путь к файлу или отключите проверку сертификата сервера, "
"изменив sslmode.\n"

#: fe-secure-openssl.c:1025
#, c-format
msgid "could not open certificate file \"%s\": %s\n"
msgstr "не удалось открыть файл сертификата \"%s\": %s\n"

#: fe-secure-openssl.c:1044
#, c-format
msgid "could not read certificate file \"%s\": %s\n"
msgstr "не удалось прочитать файл сертификата \"%s\": %s\n"

#: fe-secure-openssl.c:1069
#, c-format
msgid "could not establish SSL connection: %s\n"
msgstr "не удалось установить SSL-соединение: %s\n"

#: fe-secure-openssl.c:1103
#, c-format
msgid "could not set SSL Server Name Indication (SNI): %s\n"
msgstr ""
"не удалось задать SNI (Server Name Indication) для SSL-подключения: %s\n"

#: fe-secure-openssl.c:1149
#, c-format
msgid "could not load SSL engine \"%s\": %s\n"
msgstr "не удалось загрузить модуль SSL ENGINE \"%s\": %s\n"

#: fe-secure-openssl.c:1161
#, c-format
msgid "could not initialize SSL engine \"%s\": %s\n"
msgstr "не удалось инициализировать модуль SSL ENGINE \"%s\": %s\n"

#: fe-secure-openssl.c:1177
#, c-format
msgid "could not read private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "не удалось прочитать закрытый ключ SSL \"%s\" из модуля \"%s\": %s\n"

#: fe-secure-openssl.c:1191
#, c-format
msgid "could not load private SSL key \"%s\" from engine \"%s\": %s\n"
msgstr "не удалось загрузить закрытый ключ SSL \"%s\" из модуля \"%s\": %s\n"

#: fe-secure-openssl.c:1228
#, c-format
msgid "certificate present, but not private key file \"%s\"\n"
msgstr "сертификат присутствует, но файла закрытого ключа \"%s\" нет\n"

#: fe-secure-openssl.c:1236
#, c-format
msgid ""
"private key file \"%s\" has group or world access; permissions should be "
"u=rw (0600) or less\n"
msgstr ""
"к файлу закрытого ключа \"%s\" имеют доступ все или группа; права должны "
"быть u=rw (0600) или более ограниченные\n"

#: fe-secure-openssl.c:1261
#, c-format
msgid "could not load private key file \"%s\": %s\n"
msgstr "не удалось загрузить файл закрытого ключа \"%s\": %s\n"

#: fe-secure-openssl.c:1279
#, c-format
msgid "certificate does not match private key file \"%s\": %s\n"
msgstr "сертификат не соответствует файлу закрытого ключа \"%s\": %s\n"

#: fe-secure-openssl.c:1379
#, c-format
msgid ""
"This may indicate that the server does not support any SSL protocol version "
"between %s and %s.\n"
msgstr ""
"Это может указывать на то, что сервер не поддерживает ни одну версию "
"протокола SSL между %s и %s.\n"

#: fe-secure-openssl.c:1415
#, c-format
msgid "certificate could not be obtained: %s\n"
msgstr "не удалось получить сертификат: %s\n"

#: fe-secure-openssl.c:1521
#, c-format
msgid "no SSL error reported"
msgstr "нет сообщения об ошибке SSL"

#: fe-secure-openssl.c:1530
#, c-format
msgid "SSL error code %lu"
msgstr "код ошибки SSL: %lu"

#: fe-secure-openssl.c:1778
#, c-format
msgid "WARNING: sslpassword truncated\n"
msgstr "ПРЕДУПРЕЖДЕНИЕ: значение sslpassword усечено\n"

#: fe-secure.c:267
#, c-format
msgid "could not receive data from server: %s\n"
msgstr "не удалось получить данные с сервера: %s\n"

#: fe-secure.c:380
#, c-format
msgid "could not send data to server: %s\n"
msgstr "не удалось передать данные серверу: %s\n"

#: win32.c:314
#, c-format
msgid "unrecognized socket error: 0x%08X/%d"
msgstr "нераспознанная ошибка сокета: 0x%08X/%d"

#~ msgid "invalid channel_binding value: \"%s\"\n"
#~ msgstr "неверное значение channel_binding: \"%s\"\n"

#~ msgid "invalid ssl_min_protocol_version value: \"%s\"\n"
#~ msgstr "неверное значение ssl_min_protocol_version: \"%s\"\n"

#~ msgid "invalid ssl_max_protocol_version value: \"%s\"\n"
#~ msgstr "неверное значение ssl_max_protocol_version: \"%s\"\n"

#~ msgid "invalid gssencmode value: \"%s\"\n"
#~ msgstr "неверное значение gssencmode: \"%s\"\n"

#~ msgid "invalid target_session_attrs value: \"%s\"\n"
#~ msgstr "неверное значение target_session_attrs: \"%s\"\n"

#~ msgid ""
#~ "could not connect to server: %s\n"
#~ "\tIs the server running on host \"%s\" (%s) and accepting\n"
#~ "\tTCP/IP connections on port %s?\n"
#~ msgstr ""
#~ "не удалось подключиться к серверу: %s\n"
#~ "\tОн действительно работает по адресу \"%s\" (%s)\n"
#~ "\t и принимает TCP-соединения (порт %s)?\n"

#~ msgid "setsockopt(%s) failed: %s\n"
#~ msgstr "ошибка в setsockopt(%s): %s\n"

#~ msgid "WSAIoctl(SIO_KEEPALIVE_VALS) failed: %ui\n"
#~ msgstr "ошибка в WSAIoctl(SIO_KEEPALIVE_VALS): %ui\n"

#~ msgid "could not make a writable connection to server \"%s:%s\"\n"
#~ msgstr ""
#~ "не удалось установить подключение для чтения/записи к серверу \"%s:%s\"\n"

#~ msgid "test \"SHOW transaction_read_only\" failed on server \"%s:%s\"\n"
#~ msgstr ""
#~ "проверка \"SHOW transaction_read_only\" не пройдена на сервере \"%s:%s\"\n"

#~ msgid "function requires at least protocol version 3.0\n"
#~ msgstr "функция требует протокол минимум версии 3.0\n"

#~ msgid "COPY IN state must be terminated first\n"
#~ msgstr "сначала должно завершиться состояние COPY IN\n"

#~ msgid "COPY OUT state must be terminated first\n"
#~ msgstr "сначала должно завершиться состояние COPY OUT\n"

#~ msgid "cannot determine OID of function lo_truncate\n"
#~ msgstr "не удалось определить OID функции lo_truncate\n"

#~ msgid "cannot determine OID of function lo_truncate64\n"
#~ msgstr "не удалось определить OID функции lo_truncate64\n"

#~ msgid "cannot determine OID of function lo_lseek64\n"
#~ msgstr "не удалось определить OID функции lo_lseek64\n"

#~ msgid "cannot determine OID of function lo_create\n"
#~ msgstr "не удалось определить OID функции lo_create\n"

#~ msgid "cannot determine OID of function lo_tell64\n"
#~ msgstr "не удалось определить OID функции lo_tell64\n"

#~ msgid "cannot determine OID of function lo_open\n"
#~ msgstr "не удалось определить OID функции lo_open\n"

#~ msgid "cannot determine OID of function lo_creat\n"
#~ msgstr "не удалось определить OID функции lo_creat\n"

#~ msgid "cannot determine OID of function lo_unlink\n"
#~ msgstr "не удалось определить OID функции lo_unlink\n"

#~ msgid "cannot determine OID of function lo_lseek\n"
#~ msgstr "не удалось определить OID функции lo_lseek\n"

#~ msgid "cannot determine OID of function loread\n"
#~ msgstr "не удалось определить OID функции loread\n"

#~ msgid "cannot determine OID of function lowrite\n"
#~ msgstr "не удалось определить OID функции lowrite\n"

#~ msgid "invalid setenv state %c, probably indicative of memory corruption\n"
#~ msgstr "неверное состояние setenv %c - возможно разрушение памяти\n"

#~ msgid "invalid state %c, probably indicative of memory corruption\n"
#~ msgstr "неверное состояние %c - возможно разрушение памяти\n"

#~ msgid ""
#~ "unexpected character %c following empty query response (\"I\" message)"
#~ msgstr "неожиданный символ %c вслед за пустым ответом (сообщение \"I\")"

#~ msgid ""
#~ "server sent data (\"D\" message) without prior row description (\"T\" "
#~ "message)"
#~ msgstr ""
#~ "сервер отправил данные (сообщение \"D\") без предварительного описания "
#~ "строки (сообщение \"T\")"

#~ msgid ""
#~ "server sent binary data (\"B\" message) without prior row description (\"T"
#~ "\" message)"
#~ msgstr ""
#~ "сервер отправил двоичные данные (сообщение \"B\") без предварительного "
#~ "описания строки (сообщение \"T\")"

#~ msgid "lost synchronization with server, resetting connection"
#~ msgstr "потеряна синхронизация с сервером; попытка восстановить соединение"

#~ msgid "extraneous data in \"T\" message"
#~ msgstr "лишние данные в сообщении \"T\""

#~ msgid "extraneous data in \"t\" message"
#~ msgstr "лишние данные в сообщении \"t\""

#~ msgid "extraneous data in \"D\" message"
#~ msgstr "лишние данные в сообщении \"D\""

#~ msgid "SSL library does not support CRL certificates (file \"%s\")\n"
#~ msgstr "Библиотека SSL не поддерживает проверку CRL (файл \"%s\")\n"

#~ msgid "could not get home directory to locate password file\n"
#~ msgstr "не удалось получить домашний каталог для загрузки файла паролей\n"

#~ msgid "could not get home directory to locate service definition file"
#~ msgstr ""
#~ "не удалось получить домашний каталог для загрузки файла определений служб"

#~ msgid "setsockopt(TCP_KEEPIDLE) failed: %s\n"
#~ msgstr "ошибка в setsockopt(TCP_KEEPIDLE): %s\n"

#~ msgid "setsockopt(TCP_KEEPALIVE) failed: %s\n"
#~ msgstr "ошибка в setsockopt(TCP_KEEPALIVE): %s\n"

#~ msgid "setsockopt(TCP_KEEPINTVL) failed: %s\n"
#~ msgstr "ошибка в setsockopt(TCP_KEEPINTVL): %s\n"

#~ msgid "setsockopt(SO_KEEPALIVE) failed: %s\n"
#~ msgstr "ошибка в setsockopt(SO_KEEPALIVE): %s\n"

#~ msgid "could not acquire mutex: %s\n"
#~ msgstr "не удалось заблокировать семафор: %s\n"

#~ msgid "socket not open\n"
#~ msgstr "сокет не открыт\n"

#~ msgid "unrecognized return value from row processor"
#~ msgstr "процессор строк вернул нераспознанное значение"

#~ msgid "could not restore nonblocking mode on socket: %s\n"
#~ msgstr "не удалось вернуть сокет в неблокирующий режим: %s\n"

#~ msgid "Kerberos 5 authentication rejected: %*s\n"
#~ msgstr "аутентификация Kerberos 5 не пройдена: %*s\n"

#~ msgid "could not set socket to blocking mode: %s\n"
#~ msgstr "не удалось перевести сокет в блокирующий режим: %s\n"
